website/views/options/inventory/seasonal-shop.jade

```diff
@@ -20,17 +20,17 @@
           ng-click='buyQuest(quest.key)')
         p {{::quest.value}}&nbsp;
           span.Pet_Currency_Gem1x.inline-gems
-    // menu.pets-menu(label=env.t('seasonalItems'))
+    menu.pets-menu(label=env.t('seasonalItems'))
       div
-        button.customize-option(class='inventory_special_shinySeed',
```

Comment:
Oh duh.  I was confused because I couldn't find any other references to it, but it's because the `spritesmith-main-5.css` diff is too large to be displayed.  Thanks!